### DISCUSSION TITLE
Run Python on Seedbox rip Onlyfans

### DISCUSSION THREAD

**Discussion by Macmasteri** [UltimaHoarder/UltimaScraper#1481] [Q&A]
As the the title says. Is it possible to run python on seedbox and use DC script on it for ripping onlyfans content?

**Comment by UltimaHoarder** [comment]
Seedbox is just another VPS right? If it's got an operating system then ye it will run

**Comment by americanseeder1865** [comment]
Yes but you may have to have a way to remote in via a GUI (such as VNC) just in case you need to solve a captcha. I have tried doing this but it was not fun or easy.
